server/tracedb/awsxray.go

```diff
@@ -129,11 +129,16 @@ func parseXRayTrace(traceID string, rawTrace *xray.Trace) (model.Trace, error) {
 		return model.Trace{}, nil
 	}
 
-	segment := rawTrace.Segments[0]
-	spans, err := parseSegmentToSpans([]byte(*segment.Document), traceID)
+	spans := []model.Span{}
 
-	if err != nil {
-		return model.Trace{}, err
```

Comment:
This is fixing an issue with the x-ray integration caused by us just looking at the initial segment instead of the full list